site/content/guides/monitoring-as-code-cli.md

```diff
@@ -0,0 +1,347 @@
+
+E2E testing evolved similarly during the last decade during from proprietary algorithms living in tools like HP Quality Center to open-source-based code hosted in your repository next to your application code. Today, cross-functional DevOps teams continuously run automated E2E tests in their CI/CD pipeline instead of isolated QA teams testing new versions of your app for three months before release.
+
+## Monitoring as integral part of your SDLC
+
+Setting up monitoring can present some of the same issues as provisioning infrastructure or testing. This becomes apparent when we move past the initial rollout or proof-of-concept and onboard multiple products and/or teams, and see our monitoring setup rapidly grow in scope - along with its maintenance needs, that is.
+
+Monitoring as code learns from IaC, Unit testing and E2E testing and brings your monitoring close to your development workflows. How? By having it defined as code, much like you would do with any kind of End-to-End tests.
+
+## Why Monitoring as Code with the Checkly CLI
+
+What does one gain when moving from a manual to a monitoring as code approach? The **main advantages of the Checkly CLI** are:
+
+1. **Unite E2E testing & monitoring in one workflow.** No more silos between Dev, QA and Ops.
+2. **Programmable, testable, reviewable.** Works with your dev pipeline. From your IDE, via PR to CI.
+3. **Native @playwright/test support.** No lock-in, just write standard *.spec.ts files.
+4. **Alerting baked in.** Set alerts for Slack, SMS and many more channels.
+5. **Typescript-first.** Fully typed for a stellar developer experience with code completion.
+6. **Run in the cloud or on-prem**.** Run on the Checkly cloud or in your network using the Private Locations
+
+## Monitoring as Code with Checkly
+
+Users who have just started out will be familiar with creating checks, groups, alert channels and other resources through the Checkly UI. Creating, debugging and managing E2E and synthetic monitoring at scale is best done “as code”. Currently, Checkly supports three tools you can use for your monitoring as code (MaC) workflow:
+
+1. {{< newtabref  href="https://www.checklyhq.com/docs/cli/" title="The Checkly CLI." >}}
+2. {{< newtabref  href="https://checklyhq-com-git-cli-guide-checkly.vercel.app/docs/terraform-provider/" title="Terraform (through our Terraform provider)" >}}
+3. {{< newtabref  href="https://www.checklyhq.com/docs/integrations/pulumi/" title="Pulumi (through our Pulumi provider)" >}}
+
+## Monitoring an e-commerce website - as code
+
+How does this all look like in practice? Let's find out by creating a small monitoring setup for our {{< newtabref  href="https://danube-web.shop/" title="demo e-commerce website" >}}.
+
+### Setting up our CLI
```

Comment:
I would point to the `npm create @checkly/cli` install. It's much quicker.

Reply:
Right; Let's get the guide published and I'll update by end of the week to `npm create @checkly/cli`.